proto/p4info/Makefile.am

```diff
@@ -9,11 +9,18 @@ bin_PROGRAMS = pi_convert_p4info
 
 # convenience library to convert back-and-forth between p4info and the P4Info
 # protobuf message
-noinst_LTLIBRARIES = libpiconvertproto.la
+lib_LTLIBRARIES = libpiconvertproto.la
 
 libpiconvertproto_la_SOURCES = \
 p4info_to_and_from_proto.cpp \
-p4info_to_and_from_proto.h
+p4info_to_and_from_proto.h 
+
+#nobase_include_HEADERS = PI/proto/p4info/p4info_to_and_from_proto.h
+
+libpiconvertproto_la_LIBADD = \
```

Comment:
I may be missing something, but I think the dependency is only on protobuf, not grpc